src/Host.cpp

```diff
@@ -1147,6 +1147,38 @@ bool Host::installPackage( const QString& fileName, int module )
     }
     // reorder permanent and temporary triggers: perm first, temp second
     mTriggerUnit.reorderTriggersAfterPackageImport();
+
+    // This event was inspired by "Zoilder" in a Mudlet derivative in 2015
+    // Their version appended the packageName and "Event" to
+    // "sysInstallPackage", but I have revised it to pass the packageName as a
+    // separate argument.
+    // Due to the range of circumstances in which this method can be called
+    // I have also included:
+    // * the invoking "module" argument in case it is necesary to differentiate
+    // the way in which the installPackage code is called - I think this could
+    // be a factor in some use cases that Zoilder would not have experienced in
+    // their usage.
+    // * the original package filename
+    //
+    // The originators comment was (ES):
+    // Se ejecuta el evento que indica que se ha instalado el paquete.
+    // Así el propio paquete puede auto-inicializarse sin tener que reabrir la
+    // sesión.
+    // which I think translates to (EN):
+    // The event that indicates that the package is installed runs.
+    // Thus the package itself can auto-initialize without having to reopen the
+    // session.
+    TEvent installEvent;
+    installEvent.mArgumentList.append( QStringLiteral( "sysInstallPackageEvent" ) );
+    installEvent.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
+    installEvent.mArgumentList.append( packageName );
+    installEvent.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
+    installEvent.mArgumentList.append( QString::number( module ) );
```

Comment:
I actually don't care that much about numbers vs words, I was more pointing out that there was no legacy exposure of these numbers to users so any choice could be made. I actually think that this is symptomatic of a bad design in shoving everything into one event. I think having sys PackageInstall, sysModuleInstall, etc is the correct thing to do instead of cramming everything here